docs/definitions/parties_summary.csv

```diff
@@ -1,15 +1,15 @@
 Column Name,Data Type,Description
 id,bigint,"Unique id representing a release, compiled_release or record"
-party_index,bigint,"Unique id representing a release, compiled_release or record"
-release_type,text,"Either release, compiled_release or record. compiled_release are releases generated by kingfisher release compilation"
+party_index,bigint,Position of the party in the parties array
+release_type,text,"Either release, compiled_release or record. compiled_releases are releases generated by kingfisher release compilation"
 collection_id,integer,id from kingfisher collection table
 ocid,text,ocid from the data
-release_id,text,Release id from the data. Relevent for releases and not for compiled_releases or records
+release_id,text,Release id from the data. Relevant for releases and not for compiled_releases or records
 data_id,integer,"id for the ""data"" table in kingfisher that holds the original JSON data."
-parties_id,text,
-roles,jsonb,
-identifier,text,
-unique_identifier_attempt,text,
-parties_additionalidentifiers_ids,jsonb,
-parties_additionalidentifiers_count,integer,
-party,jsonb,
+parties_id,text,`id` in party object
+roles,jsonb,JSONB list of the party roles
+identifier,text,Concatenation of `scheme` and `id` from `identifier` object in the form `<scheme>-<id>`
+unique_identifier_attempt,text,"The `id` from party object if it exists, otherwise the identifier described above if it exists, otherwise the party name"
+parties_additionalidentifiers_ids,jsonb,JSONB list of the concatenating scheme and id of all additionalidentifier objects
```

Comment:
To be consistent with elsewhere, and to clarify, I think this is clearest:

JSONB list of the concatenation of scheme and id